README.md

```diff
@@ -1,10 +1,19 @@
 # Balta
 
-Scala library to write Postgres DB code tests with
+Balta makes testing DB entities from Scala easier. It's primarily focused on testing the behavior of [Postgres functions](https://www.postgresql.org/docs/current/xfunc.html).
+
+---
+
+### Build Status
+
+[![Build](https://github.com/AbsaOSS/balta/workflows/Build/badge.svg)](https://github.com/AbsaOSS/balta/actions)
+
+[![Maven Central](https://maven-badges.herokuapp.com/maven-central/za.co.absa.db/balta_2.12/badge.svg)](https://search.maven.org/search?q=g:za.co.absa.db.balta)
```

Comment:
Why Maven central?

Reply:
Not sure what you mean by the question. It's a link to the Maven repository for the lib.